feature/src/main/java/com/teamdontbe/feature/home/HomeFragment.kt

```diff
@@ -202,24 +209,32 @@ class HomeFragment : BindingFragment<FragmentHomeBinding>(R.layout.fragment_home
             )
             binding.rvHome.startAnimation(slideDown)
 
-            homeViewModel.getFeedList()
+            homeFeedAdapter.refresh()
+            scrollRecyclerViewToTopWhenObserveDataRefresh()
             binding.swipeRefreshLayout.isRefreshing = false
         }
     }
 
-//    private fun scrollRecyclerViewToTop() {
-//        val mainActivity = requireActivity() as? MainActivity
-//        val nestedScrollMyPage = binding.nestedScrollHome
-//
-//        mainActivity?.findViewById<BottomNavigationView>(R.id.bnv_main)
-//            ?.setOnItemReselectedListener { item ->
-//                if (item.itemId == R.id.fragment_home && nestedScrollMyPage != null) {
-//                    nestedScrollMyPage.post {
-//                        nestedScrollMyPage.smoothScrollTo(0, 0)
-//                    }
-//                }
-//            }
-//    }
+    private fun scrollRecyclerViewToTopWhenObserveDataRefresh() {
+        homeFeedAdapter.registerAdapterDataObserver(object : RecyclerView.AdapterDataObserver() {
+            override fun onItemRangeInserted(positionStart: Int, itemCount: Int) {
+                super.onItemRangeInserted(positionStart, itemCount)
+                binding.rvHome.smoothScrollToPosition(0)
+                homeFeedAdapter.unregisterAdapterDataObserver(this)
+            }
+        })
+    }
+
+    private fun scrollRecyclerViewToTop() {
+        val mainActivity = requireActivity() as? MainActivity
+
+        mainActivity?.findViewById<BottomNavigationView>(R.id.bnv_main)
+            ?.setOnItemReselectedListener { item ->
+                if (item.itemId == R.id.fragment_home) {
+                    binding.rvHome.smoothScrollToPosition(0)
```

Comment:
```suggestion
                    binding.rvHome.post {              
                    binding.rvHome.smoothScrollToPosition(0)
                }
```
p3 : post를 사용하면 뷰가 완전히 만들어 진 후에 스크롤을 하는 작업을 수행해서
조금더 안전 할 수 있다고 합니다!

Reply:
오 굿!!!

Reply:
0251c91f19d63eb68f67e3202ea35243b9e71e3c